pyseir/inference/model_fitter.py

```diff
@@ -1081,7 +1081,7 @@ def run_state(state, states_only=False, with_age_structure=False):
         ].fips.values
 
         if len(all_fips) > 0:
-            with Pool() as p:
+            with Pool(maxtasksperchild=1) as p:
```

Comment:
Definitely no need to do this now, but it'd be nice to have this as a common function so that we're getting all pools initialized with the same parameters. 

Could make a quick wrapping context manager function something like this: 
```
@contextlib.contextmanager
def get_pool(*args, **kwargs):
    with Pool(*args, maxtasksperchild=1, **kwargs) as p:
        yield p
```
would probably work?

Reply:
A common context manager holding our preferred kwargs makes sense. An alternative is reusing the same pool, passing around `p` as a function parameter where it is needed.